server/app/services/applicant/ApplicantService.java

```diff
@@ -434,12 +434,20 @@ public CompletionStage<ApplicationModel> submitApplication(
           .thenComposeAsync(
               v -> submitterProfile.getAccount(), classLoaderExecutionContext.current())
           .thenComposeAsync(
-              account ->
-                  submitApplication(
-                      applicantId,
-                      programId,
-                      /* tiSubmitterEmail= */ Optional.of(account.getEmailAddress()),
-                      request),
+              account -> {
+                // accounts are 1:1 with applicants, so we can get the first one
+                Long tiApplicantId = account.ownedApplicantIds().get(0);
```

Comment:
Conceptually this is true, but we have nothing in place to enforce it. I know I have data in prod that has multiple that needs to be cleaned up. I haven't checked for a while if it's still occurring. I think it was fixed, but I can check tomorrow.

Reply:
Would it be better to check if any of the TI's `ownedpplicantId`s are the same as the one the TI is applying for?

Reply:
Is there a better way to check if the TI is applying for themselves?

Reply:
Maybe check if the account/applicant email address are the same?

Reply:
discussed in slack, checking if the applicant ID is in the TI's `ownedApplicantIds` works.